### DISCUSSION TITLE
Disqus comments not working

### DISCUSSION THREAD

**Discussion by realnitinworks** [mmistakes/so-simple-theme#317] [Q&A] [answered]
I added my disqus shortname in _config.yml and enabled comments: true under defaults. However, I do not see the disqus loading. I have hosted the site on github pages. So I believe, I don't have to set anyother things like environment. My site is here realnitinworks.github.io

**Comment by mmistakes** [comment, marked as answer]
It's not working because you haven't set the correct layout on your post.
It should be `layout: post` not `layout: posts`.

Only `layout: post` has support for showing comments.
